src/tweens/Timeline.js

```diff
@@ -545,6 +545,22 @@ var Timeline = new Class({
         return this;
     },
 
+    /**
+     * Delegates #makeActive to the Tween manager.
+     *
+     * @method Phaser.Tweens.Timeline#makeActive
+     * @since 3.3.0
+     *
+     * @param {object} tween - The tween object to make active.
+     *
+     * @return {Phaser.Tweens.TweenManager} This Tween Manager object.
+     */
+    makeActive: function (tween)
+    {
+        this.manager && this.manager.makeActive(tween);
```

Comment:
A Timeline can never 'not have' a manager, so you could probably just single-line this to `return this.manager.makeActive(tween);`.